public/app/system.conf.js

```diff
@@ -1,6 +1,15 @@
+function getBaseURL() {
+  "use strict";
+  /* in the case of system.build, window is undefined
+     We should still pull the plugins from the CDN*/
+  if(typeof(window) === "undefined") {
+    return "public";
+  }
+  return window.grafanaBootData.settings.appStaticUrl || "public";
+}
 System.config({
   defaultJSExtenions: true,
-  baseURL: 'public',
+  baseURL: getBaseURL(),
```

Comment:
you cannot change baseURL in system.conf some things are loaded async (plugins and some sections)